docs/testfairy/app-distribution/auto-update.md

```diff
@@ -8,7 +8,14 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-Auto-update is a feature that allows you to seamlessly push new versions of your app to users, ensuring that everyone is always using the latest version. By enabling auto-update, users with older versions of the app will receive a notification prompting them to update the next time they use the app.
+Auto-update is a feature that allows you to seamlessly push new versions of 
+your app to users, ensuring that everyone is always using the latest version. By 
+enabling auto-update, users with older versions of the app will receive a 
+notification prompting them to update the next time they use the app.
+
+Every app may have a single build that is marked for "Auto Update". Builds upload 
+prior to the marked build may be updated, and builds uploads after won't 
+trigger a request of update.
 
 :::note
```

Comment:
@gmegidish-saucelabs lets remove this. super confusing. maybe call out that, do not use auto-update on the version of your app you submit to play store/ appstore